frontend/src/js/components/modals/CachedFiles/CachedFiles.jsx

```diff
@@ -68,21 +76,33 @@ class CachedFiles extends Component {
         let filename = this.state.selectedFile;
         let path = this.selectedFilePath;
 
-        return this.getProvenance(path, variable)
-            .then((result) => {
-                let var_obj = {};
-                var_obj[this.variableName] = {
-                    cdms_var_name: variable,
-                    filename: filename,
-                    path: path,
-                    provenance: result,
-                    dimension: this.dimension
-                };
-                this.props.loadVariables([var_obj])
-            })
-            .then(() => {
-                this.setState({ redefinedVariableName: '' })
-            });
+        let var_obj = {};
+        var_obj[this.variableName] = {
+            cdms_var_name: variable,
+            axisList: this.state.selectedVariable.axisList,
+            filename: filename,
+            path: path,
+            dimension: this.state.dimension
+        };
+        this.props.loadVariables([var_obj]);
+        this.setState({ redefinedVariableName: '' });
+
+        // return this.getProvenance(path, variable)
```

Comment:
This return statements is saved in the git history, do we need to keep a large section of commented code in the repo? I'm a big no commented out code type of dude. Thoughts?

Reply:
I agree. I think old code can be found in history so generally no need to inline them.
But for this one, the getProvenance was purposely created by someone else, but I explicitly not need it at this moment, I am afraid I may miss something important here. So I kept this one particularly.

Reply:
But since we talk about it, I remember this good this time, so I could remove it.